.travis.yml

```diff
@@ -32,7 +32,7 @@ before_script:
   - echo 'assert.exception=On' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
 script:
-  - ./phpunit --coverage-clover=coverage.xml
+  - ./phpunit --coverage-clover=coverage.xml --testdox --color=auto
```

Comment:
This is just to demonstrate the output, can be removed later of course

Reply:
Please do not change `.travis.yml`.